include/configuration_parser.h

```diff
@@ -62,6 +62,7 @@ class ConfigurationParser {
   inline std::shared_ptr<TiXmlHandle> XmlHandle() { return _config; }
   inline std::string getInitScriptPath() { return _init_script_path; }
   inline std::string getModelName() { return _model_name; }
+  inline int getRealtimeFactor() { return _realtime_factor; }
```

Comment:
You can drop the `inline`, declaring the method in the class already tells the compiler to inline it.

Reply:
Thanks!